lib/envelope.ts

```diff
@@ -0,0 +1,83 @@
+import { v4 as uuidv4 } from 'uuid';
+import { Address, Request, Envelope, Error, Response } from './types/lib/types';
+import crypto from 'crypto';
```

Comment:
crypto is a nodejs package, can we replace it with another package that works on the browser? (maybe crypto-js)